.github/workflows/build.yaml

```diff
@@ -109,17 +109,18 @@ jobs:
           echo '${{ toJSON(matrix.packages) }}' | jq -r '.[]' | xargs -I% nix run .#% -- --version
 
       - name: Extract branch name
+        id: extract_branch
         shell: bash
         run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
-        id: extract_branch
 
       - name: Init from template
         if: ${{ matrix.os == 'ubuntu-latest' }}
         run: |
           cd $(mktemp -d)
           echo "Fetching template from branch ${{ steps.extract_branch.outputs.branch }}"
           nix flake init -t "github:holochain/holonix/${{ steps.extract_branch.outputs.branch }}#${{ matrix.template }}"
-          nix develop -c holochain --version
+          nix develop --override-input holonix "github:holochain/holonix/${{ steps.extract_branch.outputs.branch }}" -c holochain --version
```

Comment:
The test should be that the template works with any changes defined in Holonix on the current branch. On `main` the override is redundant but still valid.